ownCloudSDK/File Handling/Checksums/SHA3/README.md

```diff
@@ -0,0 +1,136 @@
+# C implementation of SHA-3 and Keccak with Init/Update/Finalize API
+
+The purpose of this project is:
+
+* provide an API that hashes bytes, not bits
+* provide a simple reference implementation of a SHA-3 message digest algorithm, as defined in the [FIPS 202][fips202_standard] standard
+* assist developers in the [Ethereum](https://www.ethereum.org/) blockchain ecosystem by providing the Keccak function used there
+* implement the hashing API that employs the __IUF__ paradigm (or `Init`, `Update`, `Finalize` style)
+* answer the design questions, such as:
+  * what does the state for IUF look like?
+  * how small can the state be (224 bytes on a 64-bit system for a unified SHA-3 algorithm)
+  * what is the incremental cost of adding e.g. SHA3-384 to a SHA3-256 implementation?
+
+The implementation is written in C and uses `uint64_t` types to manage the SHA-3 state. The code will compile and run on 64-bit and 32-bit architectures (`gcc` and `gcc -m32` on `x86_64` were tested).
+
+[fips202_standard]: http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf "FIPS 202 standard"
+
+## License, prior work
+
+This work is licensed with a standard [MIT license](LICENSE). I appreciate, but do not require, any attribution to this work if you used the code or ideas. I thank you for this in advance.
```

Comment:
@felix-schwarz will our sbom tool detect this dependency? Would it be possible to pull this dep via the xcode dependency management? THX

Reply:
We need to know the location from where this was taken including version information. THX

Reply:
It is from https://github.com/brainhub/SHA3IUF.git and I'm working on integrating it as SwiftPM and/or get it into a place that cdxgen should recognize.

Reply:
@felix-schwarz will this be included into this PR?